presto-native-execution/presto_cpp/main/PeriodicTaskManager.cpp

```diff
@@ -678,25 +678,31 @@ void PeriodicTaskManager::addWatchdogTask() {
         RECORD_METRIC_VALUE(kCounterNumTasksDeadlock, deadlockTasks.size());
         for (const auto& call : stuckOpCalls) {
           LOG(ERROR) << "Stuck operator: tid=" << call.tid
-                     << " taskId=" << call.taskId << " opId=" << call.opId;
+                     << " taskId=" << call.taskId << " opCall=" << call.opCall
+                     << " duration= " << velox::succinctMillis(call.durationMs);
         }
         RECORD_METRIC_VALUE(kCounterNumStuckDrivers, stuckOpCalls.size());
         if (!deadlockTasks.empty() || !stuckOpCalls.empty()) {
           detachWorker();
+        } else {
+          maybeAttachWorker();
         }
       },
       60'000'000, // 60 seconds
       "Watchdog");
 }
 
 void PeriodicTaskManager::detachWorker() {
```

Comment:
Any reason this method name doesn't have 'maybe' prefix like maybeAttachWorker?

Reply:
Which method? You highlighted the 'maybeAttachWorker'.

Reply:
I was asking why detachWorker is not named maybeDetachWorker.

Reply:
@mbasmanova 

As I mentioned in another comment (above):

Technically, detach wouldn't detach if we are already detached.
But the end result would be 'detached'.

Attach, on the other hand, wouldn't attach if we are detached, because we are shutting down.

That's the difference.

Reply:
I see. I didn't understand it until now.

I wonder if other readers will be confused as well. Would it be possible to update comments to make things a bit clearer?